node/generate-with-together-ai/static/index.html

```diff
@@ -0,0 +1,90 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Replicate Demo</title>
```

Comment:
```suggestion
  <title>Together AI Demo</title>
```
Please run a search through the entire folder to make sure we dont make this mistake in other places